sqlglot/dialects/bigquery.py

```diff
@@ -333,3 +333,6 @@ def intersect_op(self, expression: exp.Intersect) -> str:
 
         def with_properties(self, properties: exp.Properties) -> str:
             return self.properties(properties, prefix=self.seg("OPTIONS"))
+
+        def parameter_sql(self, expression: exp.Parameter) -> str:
+            return f"{self.PARAMETER_TOKEN}`{expression.this.this}`"
```

Comment:
can you show the documentation that this is required?

Reply:
The backticks are not necessary, and I'm not sure if this is always correct. For example, in https://cloud.google.com/bigquery/docs/parameterized-queries there's a query where you can omit them:

```
bq query \
   --use_legacy_sql=false \
   --parameter=corpus::romeoandjuliet \
   --parameter=min_word_count:INT64:250 \
   'SELECT
     word,
     word_count
   FROM
     `bigquery-public-data.samples.shakespeare`
   WHERE
     corpus = @corpus
   AND
     word_count >= @min_word_count
   ORDER BY
     word_count DESC;'
```